piplicenses.py

```diff
@@ -296,6 +341,10 @@ def create_parser():
                         action='store_true',
                         default=False,
                         help='dump as json')
+    parser.add_argument('--summary',
+                        action='store_true',
+                        default=False,
+                        help='show license use summary only')
```

Comment:
Can you change it like this?: `help='dump summary of each license'`

Reply:
Sure thing.